src/routes/associate/Game/Game.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { BrowserView, MobileView } from 'react-device-detect';
+import { DndProvider } from 'react-dnd';
```

Comment:
React dnd does not work on mobile. If cannot figure out, I will take over when you change the layout of the game

Reply:
Noted, will continue working on the previously implement container using react-draggable